src/app/legacy/components/Promo/image.jsx

```diff
@@ -55,7 +59,7 @@ const Image = props => {
     <Wrapper>
       <IMAGE
         {...rest}
-        src={src.replace('{width}', 240)}
+        src={src.replace('{width}', 660)}
```

Comment:
Think you want lowest to appear here by default.

Reply:
Was just testing it out while it was in draft!

Reply:
very eager @eagerterrier 👀